src/lecturer/components/notifiableTab/NotifiableTab.tsx

```diff
@@ -21,21 +22,22 @@ const StyledBadge = withStyles((theme: Theme) =>
     })
 )(Badge);
 export default function NotifiableTab(props: NotifiableTabProps) {
-
+    const nothing = () =>{}
     const [notifiableNumber, setNotifiableNumber] = useState(0);
     const label = props.label ?? "none";
     const routes = props.routes ?? "";
     const value = props.value ?? "";
+    const resetFunction = props.resetFunction?? nothing;
```

Comment:
```suggestion
    const [notifiableNumber, setNotifiableNumber] = useState(0);
    const label = props.label ?? "none";
    const routes = props.routes ?? "";
    const value = props.value ?? "";
    const resetFunction = props.resetFunction ?? (() =>{ });
```
This is much cleaner